src/Ryujinx.Cpu/AddressSpace.cs

```diff
@@ -163,17 +165,50 @@ public AddressSpace(MemoryBlock backingMemory, ulong asSize, bool supports4KBPag
                 _privateTree = new IntrusiveRedBlackTree<PrivateMapping>();
                 _treeLock = new object();
 
-                _mappingTree.Add(new Mapping(0UL, asSize, MappingType.None));
-                _privateTree.Add(new PrivateMapping(0UL, asSize, default));
+                _mappingTree.Add(new Mapping(0UL, addressSpaceSize, MappingType.None));
+                _privateTree.Add(new PrivateMapping(0UL, addressSpaceSize, default));
             }
 
             _backingMemory = backingMemory;
             _supports4KBPages = supports4KBPages;
 
+            Base = baseMemory;
+            Mirror = mirrorMemory;
+            AddressSpaceSize = addressSpaceSize;
+        }
+
+        public static bool TryCreate(out AddressSpace addressSpace, MemoryBlock backingMemory, ulong asSize, bool supports4KBPages)
```

Comment:
```suggestion
        public static bool TryCreate(MemoryBlock backingMemory, ulong asSize, bool supports4KBPages, out AddressSpace addressSpace)
```
Try methods usually have the output parameter at the end.